pom.xml

```diff
@@ -199,14 +197,14 @@
     <!-- org.apache.commons/commons-pool2/-->
     <commons-pool2.version>2.12.0</commons-pool2.version>
     <datanucleus-core.version>4.1.17</datanucleus-core.version>
-    <guava.version>14.0.1</guava.version>
+    <guava.version>33.0.0-jre</guava.version>
```

Comment:
@pan3793 If we upgrade the version of Guava, and it is also packaged into the `newwork-shuffle module` through the shaded+relocation method, then we need to package `com.google.guava:failureaccess` at the same time.

Reply:
We actually cannot avoid exposing vanilla Guava to classpath, except to Spark internal usage, 3-rd party libs still require Guava to work, e.g. Curator, Hive. Then, should we continue to do shading after upgrading?